ple.lua

```diff
@@ -1051,17 +1051,18 @@ local function editor_loadinitfile()
 	-- function to be executed before entering the editor loop
 	-- could be used to load a configuration/initialization file
 	local initfile = os.getenv("PLE_INIT")
+	local env = {editor = editor, table.unpack(_G)}
```

Comment:
This is a bit wasted effort in case no initfile is found. Also there is some duplication on the lines below (1056, 1060, 1065).

Let me know if you prefer to also separate the locating and the loading of the initfile as in 82802371400868d148940e22c38cdb974b269e5c (addresses the wasted effort and duplication).